WMF Framework/Significant Events Endpoint/ArticleAsLivingDocViewModels.swift

```diff
@@ -844,10 +854,96 @@ public extension ArticleAsLivingDocViewModel.Event.Large {
         return attributedString
     }
     
+    static let padding = UIEdgeInsets(top: 17, left: 15, bottom: 17, right: 15)
+    static let sideScrollingCellWidth = CGFloat(250)
+    static let availableSideScrollingCellWidth = {
+        return sideScrollingCellWidth - padding.left - padding.right
+    }()
+    
+    static let changeDetailDescriptionTextStyle = DynamicTextStyle.subheadline
+    static let changeDetailDescriptionTextStyleItalic = DynamicTextStyle.italicSubheadline
+    static let changeDetailDescriptionFontWeight = UIFont.Weight.regular
+    static let changeDetailReferenceTitleStyle = DynamicTextStyle.semiboldSubheadline
+    static let changeDetailReferenceTitleDescriptionSpacing = CGFloat(13)
+    static let additionalPointsForShadow = CGFloat(16)
+    
+    func calculateTallestChangeDetailHeightForTraitCollection(_ traitCollection: UITraitCollection) -> CGFloat {
+        
```

Comment:
This function generally makes sense when I read through it, though I don't understand every lasts detail. Pulling out some logic into additional well-named functions may help with readability, but for an experiment probably isn't worthwhile.

Reply:
Agreed, it's pretty messy. I'll do a little cleanup in my sizing pass.

Reply:
I cleaned this up in [f5230a](https://github.com/wikimedia/wikipedia-ios/pull/3683/commits/f5230a814d250b649ae1d94f0798b5fa1813306e). Hopefully it's a little easier to read now.

Reply:
This looks great, thank you!